apiary-metastore-events/README.md

```diff
@@ -0,0 +1,17 @@
+# Apiary Metastore Events
+
+##  Overview
+The Apiary Metastore Events contains set of modules responsible for retrieving and processing of the Hive Metastore events.
+
+Currently, the following modules are defined:
+ - [apiary-metastore-listener](apiary-metastore-listener) - Listens to the events from Hive Metastore and publishes them to SNS
```

Comment:
Listens to the events from Hive Metastore -> Listens to events from the Hive Metastore